canonical-web-and-design/ubuntu.com
# Remove image builder

## Done
- Removed image builder and all its dependencies

## QA
- Go to /core/build and see its deleted
- Check site login still works
- Check the /advantage section still looks ok

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/3088
